openpype/hosts/maya/plugins/create/create_unreal_skeletalmesh.py

```diff
@@ -26,7 +26,7 @@ def get_dynamic_data(
         dynamic_data = super(CreateUnrealSkeletalMesh, cls).get_dynamic_data(
             variant, task_name, asset_id, project_name, host_name
         )
-        dynamic_data["asset"] = legacy_io.Session.get("AVALON_ASSET")
+        dynamic_data["asset"] = get_current_asset_name()
```

Comment:
Isn't this data in the `create_context`? Or isn't that available during `get_dynamic_data`?

Reply:
This is maya which still uses legacy create so does not have `create_context`.